src/main/java/org/skriptlang/skript/bukkit/tags/TagModule.java

```diff
@@ -0,0 +1,45 @@
+package org.skriptlang.skript.bukkit.tags;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.classes.ClassInfo;
+import ch.njol.skript.classes.Parser;
+import ch.njol.skript.lang.ParseContext;
+import ch.njol.skript.registrations.Classes;
+import org.bukkit.Tag;
+
+import java.io.IOException;
+
+public class TagModule {
+	public static void load() throws IOException {
+		// abort if no class exists
+		if (!Skript.classExists("org.bukkit.Tag"))
+			return;
+
+		// load classes (todo: replace with registering methods after regitration api
+		Skript.getAddonInstance().loadClasses("org.skriptlang.skript.bukkit", "tags");
+
+		// Classes
+		Classes.registerClass(new ClassInfo<>(Tag.class, "minecrafttag")
```

Comment:
is there no issue with registering the syntax before the classinfo?

Reply:
the patterns aren't parsed until parsing actually starts, so there should be no issues